pkg/skaffold/schema/defaults/defaults.go

```diff
@@ -102,127 +108,143 @@ func Set(c *latestV1.SkaffoldConfig) error {
 	}
 
 	setDefaultTestWorkspace(c)
+	SetDefaultRenderer(c)
```

Comment:
yup, I'll make a quick change to fix this

Reply:
@gsquared94 since the `defaults` package is meant to operate on one config at a time, I think this change will require a bit of refactoring/discussion about how we want to handle this. Do we want to do this here? 

I do think the calls to `defaults.SetDefaultRenderer()` and `defaults.SetDefaultDeployer()` in `runner.go` are redundant tho, so I'll remove them